html/manifest/background_color.json

```diff
@@ -0,0 +1,59 @@
+{
+  "progressivewebapps": {
```

Comment:
Since we've decided to move this in the `html/` folder for now, the export key "`progressivewebapps`" won't work here. Please change this to `"html"` throughout.

Reply:
Thanks for this comment.